src/java.base/share/classes/java/util/concurrent/Executors.java

```diff
@@ -391,44 +388,19 @@ public static ThreadFactory defaultThreadFactory() {
     }
 
     /**
-     * Returns a thread factory used to create new threads that
-     * have the same permissions as the current thread.
+     * Returns a thread factory used to create new threads.
      * This factory creates threads with the same settings as {@link
-     * Executors#defaultThreadFactory}, additionally setting the
-     * AccessControlContext and contextClassLoader of new threads to
-     * be the same as the thread invoking this
-     * {@code privilegedThreadFactory} method.  A new
```

Comment:
We no longer state that the contextClassLoader is set, though `PrivilegedThreadFactory` _does_ still set the ccl. Is this OK?